csharp/ql/test/library-tests/ir/ir/stmts.cs

```diff
@@ -68,9 +68,15 @@ public static void tryCatchFinally()
     public static void forStmt() 
     {
         int x = 0;
-        for (int i = 0; i <= 10; i++) 
+        for (int i = 0, j = 10; i < j; i++, j--)
+        {   
+            x = x - 1; 
+        }
+        
+        int a = 0, b = 10;
+        for (; a < b; ) 
         {
-            x = x - 1;
+            a++;
```

Comment:
I suggest adding a test for
```
for ( ; ; )
{
    ....
}
```
as well.

Reply:
Added the test case, seems to produce correct code.